### ISSUE TITLE
Use (i) instead of Ξ for info button on tracks

### ISSUE THREAD

**Issue by echarlie** [wuvt/pload#37]
hamburger implies there are actions that can be taken: a circled info button seems better

**Comment by mutantmonkey** [comment]
Well, I mean, I was thinking we might add some actions to that track info modal. That being said, what icon would you prefer that we use? https://useiconic.com/open

By the way, I tried using oi-info already and it looked out of place because that character is significantly narrower than the trash can. I also have never seen a button that looks like that anywhere, so it was a bit confusing to me.

**Comment by echarlie** [comment]
I'd prefer to use utf-8 over an icon font: https://www.utf8icons.com/character/128712/circled-information-source

**Comment by mutantmonkey** [comment]
Ah so you're saying you want 🛈 then, I see.

**Comment by echarlie** [comment]
yes. I don't have an easy way to type 🛈 with the compose key, and I had forgotten about ctrl-shift-U to type unicode in xorg.

**Comment by echarlie** [comment]
@mutantmonkey any reason that the commit resolving this hasn't been merged?

**Comment by mutantmonkey** [comment]
The "needs-user-feedback" label on this issue is the reason I haven't opened a PR yet

**Comment by echarlie** [comment]
@JasonDavidoff85 we need user feedback. Do you mind asking users? We should probably generate some screenshots for lazy A/B testing; @mutantmonkey can you do that?
